### PR TITLE
Updated sugargame and miscellaneous fixes

### PR DESCRIPTION
**Pull request by yell0wfl4sh**
- added .gitignore
- removed score.pkl
- fix fail to start
- updated sugargame

@quozl please review!

### CODE REVIEW

File: activity.py [hunk 1]
```diff
@@ -20,12 +21,15 @@ def __init__(self, handle):
...
+        self.game.canvas.grab_focus()        
         self.build_toolbar()
-        self._pygamecanvas = sugargame.canvas.PygameCanvas(self)
-        self.set_canvas(self._pygamecanvas)
-        self._pygamecanvas.grab_focus()
-        self._pygamecanvas.run_pygame(self.actividad.main)
+        #self._pygamecanvas = sugargame.canvas.PygameCanvas(self)
+        #self.set_canvas(self._pygamecanvas)
+        #self._pygamecanvas.grab_focus()
+        #self._pygamecanvas.run_pygame(self.game.make)
```
Comment (quozl) [inline activity.py]: Remove these lines rather than comment them.

File: activity.py [hunk 2]
```diff
@@ -20,12 +21,15 @@ def __init__(self, handle):
         activity.Activity.__init__(self, handle)
         self.max_participants = 1
         self.sound = True
-        self.actividad = main.game()
+        self.game = main.game()
+        self.game.canvas = sugargame.canvas.PygameCanvas(self, main=self.game.make, modules=[pygame.display, pygame.font])
```
Comment (quozl) [inline activity.py]: Wrap line; use flake8 to verify.

**Review by quozl — CHANGES_REQUESTED** [review]
Thanks, looks good, a few changes requested;

**Comment by yell0wfl4sh** [comment]
@quozl Done both as suggested.

**Pull request merged**